### ISSUE TITLE
idream -getting poor results 

### ISSUE THREAD

**Issue by orydatadudes** [justinpinkney/stable-diffusion#41]
HI
thank you for idream implementation
i used DeepFashion dataset to check if i can use change human pose ( like the paper show )
unfortunately i didn't succeed to get good generated images
for example :
source image 
![MEN_Denim_id_00000089_3_28_1_front](https://user-images.githubusercontent.com/64726228/199429879-d3ca43fc-4648-4b8a-ac31-327b585b55f2.jpg)

after trying to change  posture (“A person with crossed arms”)

 i got bad results ( for different levels of alpha )

 0.6:
![generated_image_MEN_Denim_id_00000089_3_28_1_front jpg_0 6](https://user-images.githubusercontent.com/64726228/199430622-46744704-ab84-41f6-bef9-775dd70eaabe.png)

0.8
![generated_image_MEN_Denim_id_00000089_3_28_1_front jpg_0 8](https://user-images.githubusercontent.com/64726228/199430761-16cb9592-43a1-4121-9f7f-1a8f36f5e9e2.png)

0.9:
![generated_image_MEN_Denim_id_00000089_3_28_1_front jpg_0 9](https://user-images.githubusercontent.com/64726228/199430819-9142ea26-047b-4cd0-86c6-04c08e623cc7.png)

i tried with more images and more text prompt and increase alpha range but nothing works for me
did someone succeed getting good results ?
thanks

**Comment by justinpinkney** [comment]
Yeah it seems quite sensitive to parameters, I found sometimes that doing the first stage training, (training the embedding) for longer helps.
